ui/app/clinical/consultation/controllers/bacteriologyController.js

```diff
@@ -55,7 +55,7 @@ angular.module('bahmni.clinical')
             };
 
             $scope.$on('$stateChangeStart', function () {
-                if ($scope.bacteriologyForm && $scope.bacteriologyForm.$dirty) {
+                if ($scope.bacteriologyForm.$dirty) {
```

Comment:
can we have $scope.bacteriologyForm && $scope.bacteriologyForm.$dirty check as part of the condition